common/static/coffee/src/discussion/discussion_module_view.coffee

```diff
@@ -97,7 +97,7 @@ if Backbone?
       else
         @$el.append($discussion)
 
-      @newPostForm = $('.new-post-article')
+      @newPostForm = this.$el.find('.new-post-article')
```

Comment:
I'm not clear how this fix by itself makes the click handlers properly scoped (though we should definitely never do what was on line 100 previously). Was it before creating a list of views instead of a single view?